### DISCUSSION TITLE
Multiple subscriptions with multiple workers

### DISCUSSION THREAD

**Discussion by ruem2802** [fastapi/fastapi#6605] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi_mqtt import FastMQTT, MQTTConfig

app = FastAPI()

mqtt_config = MQTTConfig()

mqtt = FastMQTT(
    config=mqtt_config
)

mqtt.init_app(app)



@mqtt.on_connect()
def connect(client, flags, rc, properties):
    mqtt.client.subscribe("/mqtt") #subscribing mqtt topic
    print("Connected: ", client, flags, rc, properties)

@mqtt.on_message()
async def message(client, topic, payload, qos, properties):
    print("Received message: ",topic, payload.decode(), qos, properties)

@mqtt.on_disconnect()
def disconnect(client, packet, exc=None):
    print("Disconnected")

@mqtt.on_subscribe()
def subscribe(client, mid, qos, properties):
    print("subscribed", client, mid, qos, properties)
```

### Description

I use fastpi-mqtt and subcribe a topic (mqtt)
I run command: uvicorn app:app --host 0.0.0.0 --port 8000 --workers 4
=>  creates 4 worker processes for me
I have 4 subscriptions 
I want to have only one subsription .What is the solution here?

### Operating System

Linux

### Operating System Details

Linux

### FastAPI Version

0.68.1

### Python Version

3.9.7

**Comment by Kludex** [comment]
It doesn't make sense, but you can check the `os.current_process().name`.

**Comment by ruem2802** [comment]
@Kludex thank for support!

**Comment by ruem2802** [comment]
@sabuhish  can you help me about that
